tutorials/3d/global_illumination/introduction_to_global_illumination.rst

```diff
@@ -121,7 +121,7 @@ In order of performance from fastest to slowest:
   - ReflectionProbes with their update mode set to **Always** are much more
     expensive than probes with their update mode set to **Once** (the default).
     Suited for integrated graphics when using the **Once** update mode.
-    *Available when using the Forward Mobile backend. Will be available in the Compatibility backend in later releases.*
+    *Available when using the Mobile and Compatibility renderers.*
```

Comment:
Are reflection probes unavailable in `Forward+`? Otherwise I'd remove this as it is available in all renderers

Reply:
That was a copy-paste error. It's available in all three renderers, but you're right it makes sense to leave out the note in that case.

Reply:
Nevermind. The conventions in this section are weird. All global illumination methods are available in Forward+, so the other notes here only describe whether the global illumation method is available in Mobile or Compatibility, for example:
`**Not available** *when using the Forward Mobile or Compatibility backends.*`
`*Available when using the Forward Mobile backend. Will be available in the Compatibility backend in later releases.*`
`**Not available** *when using the Forward Mobile or Compatibility backends.*`

With that context, the note I wrote fits local conventions.

I'm working on this section anyway as part of changes for https://github.com/godotengine/godot-docs/issues/10139, changing "Forward Mobile" to "Mobile". I'll change all these notes to be more sensible as part of that change, in another PR.

Reply:
In the case of global illumination or antialiasing, I think it does make sense to note when a method is available in all renderers, because most methods are not. That *is* a change from what these two pages currently do.

I'll make this change and change the others for consistency.